client/state/invites/reducer.js

```diff
@@ -52,4 +55,29 @@ export const items = createReducer(
 	}
 );
 
-export default combineReducers( { requesting, items } );
+/**
+ * Returns the updated site invites resend requests state after an action has been
+ * dispatched. The state reflects an object keyed by site ID, consisting of requested
+ * resend invite IDs, with a boolean representing request status.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export function requestingInviteResend( state = {}, action ) {
```

Comment:
I slightly prefer `state.invites.requestingResend` over the current `state.invites.requestingInviteResend`.